src/sass/extends/_backgrounds.scss

```diff
@@ -91,11 +91,11 @@
     opacity: 1;
 
     &-white {
-      background-image: $topo-black;
+      background-image: $topo-white;
```

Comment:
This change will effect the output of the UDS Background Section block. Honestly, these image files have changed names so many times that I am confused as to which topo-map image file you're supposed to get if you select "white topo map." =)